crates/db_schema/src/schema.rs

```diff
@@ -65,7 +65,7 @@ diesel::table! {
 }
 
 diesel::table! {
-    use diesel::sql_types::{Bool, Int4, Nullable, Text, Timestamp, Varchar};
```

Comment:
Unsure about this and similar changes in this file. They were auto generated... Should I revert them?

Reply:
Yep and also `diesel migration redo`.

Reply:
I have reverted those lines in this PR 👍